ledger/participant-state/kvutils/src/test/suite/scala/com/daml/ledger/participant/state/kvutils/KVUtilsPackageSpec.scala

```diff
@@ -63,6 +63,19 @@ class KVUtilsPackageSpec extends WordSpec with Matchers with BazelRunfiles {
       }
     }
 
+    "be able to pre-execute model-test.dar" in KVTest.runTest {
+      for {
+        // NOTE(JM): 'runTest' always uploads 'simpleArchive' by default.
```

Comment:
Why is this note relevant here? I think it should be removed (along with the above note) as it doesn't make sense in this context.

Reply:
I guess it explains why we're uploading more packages but then again it can be removed from both places.

Reply:
Done.